parm/config/gfs/config.atmanlinit

```diff
@@ -6,5 +6,6 @@
 echo "BEGIN: config.atmanlinit"
 
 # Get task specific resources
+. "${EXPDIR}/config.atmanl"
 . "${EXPDIR}/config.resources" atmanlinit
```

Comment:
You've split the comment from the line it applies to. Plus, I think our other config files all source `config.resources` first. So unless you have to source `atmanl` before resources, I would move it after. If you do need to source it first, move it before the comment. (Also applies to the other config files in this PR.)
```suggestion
# Get task specific resources
. "${EXPDIR}/config.resources" atmanlinit

. "${EXPDIR}/config.atmanl"
```

Reply:
`atmanlinit` in `config.resources` contains
```
  "atmanlinit")
    export layout_x=${layout_x_atmanl}
    export layout_y=${layout_y_atmanl}
```

`config.atmanl` defines the above layout variables as user configurable input
```
export layout_x_atmanl=@LAYOUT_X_ATMANL@
export layout_y_atmanl=@LAYOUT_Y_ATMANL@
```
This is why we source `config.atmanl` before `config.resources`.

1b289bf9 moves the `Get task specific resources` comment prior to the `config.resources` line.  A new comment, `Get task specific layout`, is added before `config.atmanl`.  Similar changes made to the JEDI ATM Ens config files.

Reply:
@WalterKolczynski-NOAA , let me know if the above is acceptable and I will resolve this conversation. Otherwise, we'll keep dialoguing until the config files are acceptable.

Reply:
Works for me.

Reply:
@RussTreadon-NOAA 
Apologies for bringing this up again.
https://github.com/NOAA-EMC/global-workflow/blob/a3374607d01fbdabbec0660afb82b5eb3677b4af/jobs/JGLOBAL_ATM_ANALYSIS_INITIALIZE#L5
sources `atmanl` first and then `atmanlinit`.  So, why does one need to source `config.atmanl` in `config.atmanlinit`?  The `layout_x_atmanl` will already be in the env. when `config.resources` is sourced for `atmanlinit`.

Reply:
When 
```
# Get task specific layout
. "${EXPDIR}/config.atmanl"
```
is not in `config.atmanlrun`, `workflow/create_experiment.py` creates an xml with 0 nodes for `atmanlrun`.  

I commented out the above `config.atmanl` in `config.atmanlrun` and ran `workflow/create_experiment.py`.  The below xml was generated for `gdasatmanlrun`
```
<task name="gdasatmanlrun" cycledefs="gdas" maxtries="&MAXTRIES;">

        <command>/work2/noaa/da/rtreadon/git/global-workflow/test/jobs/rocoto/atmanlrun.sh</command>

        <jobname><cyclestr>prtest_debug_gdasatmanlrun_@H</cyclestr></jobname>
        <account>da-cpu</account>
        <queue>batch</queue>
        <partition>orion</partition>
        <walltime>00:30:00</walltime>
        <nodes>0:ppn=40:tpp=1</nodes>
        <memory>96GB</memory>
        <native>--export=NONE</native>
```
The atmanlinit xml is OK without 
```
# Get task specific layout
. "${EXPDIR}/config.atmanl"
```
in `config.atmanlinit`

The `config.atmanl` line was added to `config.atmanlinit` to be consistent with `config.atmanlrun`.  Both the init and run jobs use `layout_x` and `layout_y`

The same comments apply to `config.atmensanlinit` and `config.atmensanlrun`.  

This is a g-w issue. I'm open to suggestions from the g-w team.

Reply:
Change committed at [`563a438`](https://github.com/RussTreadon-NOAA/global-workflow/commit/563a43826117b0fddff23b42001b4f11f6bdbd85)